_codux/boards/hero-flex.board.tsx

```diff
@@ -28,6 +28,6 @@ export default createBoard({
         </div>
     ),
     environmentProps: {
-        windowWidth: 430,
+        windowWidth: 360,
```

Comment:
Looks like an unrelated change.

Reply:
to remove it from here, this is the right thing to do? or just put back the 430 
thanks 

<img width="263" alt="image" src="https://github.com/user-attachments/assets/4c06d798-6605-4b08-a805-20ff7ad80cc9">

Reply:
I think if you click "Delete file" it will actually delete the file, not just the change. It's easier to put the value back, or if the new value is OK then just keep it.

I just wasn't sure if it's an accidental change or not.

Reply:
probably just played with it - so accidental. I'll change it back, thanks.